user-guide/Advanced_Functionality/Databases/Indexing_database/Elasticsearch/storage_per_DMA/Configuring_DataMiner_Indexing.md

```diff
@@ -17,6 +17,9 @@ The following settings are available:
 
 - *Migrate booking data to Indexing Engine*: Starts a wizard that allows you to migrate booking data from the Cassandra database to the indexing database. Only displayed in case booking data have not been migrated yet.
 
+  > [!NOTE]
+  > From DataMiner 10.3.0 [CU15]/10.4.0 [CU3]/10.4.6 onwards<!--RN 39173-->, the Indexing Engine settings are no longer available if you are using a [DaaS system](xref:Creating_a_DMS_in_the_cloud).
```

Comment:
I assume this is because these settings can't be customized for DaaS, so maybe we should rephrase the note to make that more clear - so that people with older DaaS systems will also know this is not for them.

Reply:
Indeed, local database settings are made irrelevant for the user when using DaaS.